src/router.rs

```diff
@@ -11,13 +12,20 @@ use iron::Url;
 use recognizer::Router as Recognizer;
 use recognizer::{Match, Params};
 
+
+pub struct RouterInner {
```

Comment:
Since this is a public type I don't love the name `RouterInner` how about `RouterData` or something like that?